src/view/components/customerIdWrapper.jsx

```diff
@@ -0,0 +1,170 @@
+import "regenerator-runtime"; // needed for some of react-spectrum
+import React from "react";
+import { FieldArray } from "formik";
+import Textfield from "@react/react-spectrum/Textfield";
+import Checkbox from "@react/react-spectrum/Checkbox";
+import Select from "@react/react-spectrum/Select";
+import FieldLabel from "@react/react-spectrum/FieldLabel";
+import Button from "@react/react-spectrum/Button";
+import Well from "@react/react-spectrum/Well";
+import Heading from "@react/react-spectrum/Heading";
+import Delete from "@react/react-spectrum/Icon/Delete";
+import "@react/react-spectrum/Form"; // needed for spectrum form styles
+import PropTypes from "prop-types";
+import WrappedField from "./wrappedField";
+import authenticatedStateOptions from "../constants/authenticatedStateOptions";
+import getDefaultCustomerId from "../utils/getDefaultCustomerId";
+import InfoTipLayout from "./infoTipLayout";
+import getInstanceOptions from "../utils/getInstanceOptions";
+
+function CustomerIdWrapper({ values, initInfo }) {
+  // TODO: implement in the future
```

Comment:
What is this code for? Can we remove it for now?

Reply:
I think it is for fetching the namespaces in future. We decided to keep the textfeild to begin with

Reply:
Removed for now.